build.cake

```diff
@@ -250,9 +250,9 @@ Task("TestCore")
 
     foreach (var testProject in testProjects)
     {
-        var logFile = System.IO.Path.Combine(logFolder, $"{testProject}-core-result.xml");
+        var logFile = System.IO.Path.Combine(logFolder, $"{testProject}-core-result.trx");
```

Comment:
TRX is the file extension for mstest output, which is now the dotnet test output format.